nodejs/lancedb/table.ts

```diff
@@ -226,6 +226,17 @@ export abstract class Table {
     column: string,
     options?: Partial<IndexOptions>,
   ): Promise<void>;
+
+  /**
+   * Drop an index from the table.
+   *
+   * @param name The name of the index.
+   *
+   * @note This does not delete the index from disk, it just removes it from the table.
+   * To delete the index, run {@link Table#optimize} after dropping the index.
+   */
```

Comment:
In `pylance` I added a note about how to find the `name` of the index.  Do you think we want that here?  I agree the note on deletion is a good addition.

Reply:
Ah that would probably be good to include. I will add it.